docs/RegularCheck.md

```diff
@@ -1,113 +1,73 @@
 # What you might want to check on a regular basis
 
-Most of the components deployed by scalar-kubernetes are self-healing with the help of the managed k8s services and k8s self-healing capability. There are also configured alerts that occur when some unexpected behavior happens. Thus, there shouldn't be so many things to do day by day for running a service with scalar-kubernetes. However, it is recommended to check the status of a system on a regular basis to see if everything is working fine. Here is the list of things you might want to do on a regular basis.
+Most of the components deployed by manual deployment guides are self-healing with the help of the managed Kubernetes services and Kubernetes self-healing capability. There are also configured alerts that occur when some unexpected behavior happens. Thus, there shouldn't be so many things to do day by day for the deployment of Scalar products on the managed Kubernetes cluster. However, it is recommended to check the status of a system on a regular basis to see if everything is working fine. Here is the list of things you might want to do on a regular basis.
 
-## Kubernetes
+## Kubernetes resources
 
 ### Check if Pods are all healthy statues
 
-Please check the kubernetes namespaces:
+Please check the Kubernetes namespaces:
 
-* `default` for Scalar DL
-* `logging` for Fluent bit
-* `monitoring` for Prometheus
+* `default` (or specified namespace when you deploy Scalar products) for the Scalar product deployment
+* `monitoring` for the Prometheus Operator and Loki
 
 What to check:
 
-* `STATUS` is all `Running`
-* `RESTARTS` number is all 0
+* `STATUS` is all `Running` (ScalarDL Schema Loader is `Completed`)
```

Comment:
(ScalarDL Schema Loader is `Completed`) should be removed or modified since the following shows the information about scalardb pods?

Reply:
@feeblefakie  
Thank you for your review!

I only described ScalarDB's example since the pod name is not important point in this section (the important thing is pod status) and it is redundant if we describe all examples of ScalarDB, Ledger, and Auditor (basically only the pod names are different).

And, I covered the ScalarDL Schema Loader case with this note, but it is a bit confusing.
Also, this document explains what we should check on a **regular basis**.
However, we only need to check the status of the ScalarDL Schema Loader once when we deploy it.
In other words, we don't need to check the status of the Schema Loader on a regular basis.

So, I removed this note.